website/docs/r/backup_policy_vm.html.markdown

```diff
@@ -166,6 +168,20 @@ The `retention_yearly` block supports:
 
 ---
 
+A `tiering_policy` block supports the following:
+
+* `archived_rp` - (Required) An `archived_rp` block as defined below.
+
+---
+
+An `archived_rp` block supports the following:
```

Comment:
what does rp stand for? should it be remove or expanded into what it means?

Reply:
“RP” stands for Restore Point.

Reply:
ty